src/status_im/utils/datetime.cljs

```diff
@@ -121,37 +113,41 @@
     (t/within? start end datetime)))
 
 ;;;; Timestamp formatters
-
-(defn- to-str [ms old-fmt-fn yesterday-fmt-fn today-fmt-fn]
-  (let [date (from-long ms)
-        local (plus date time-zone-offset) ; this is wrong, it uses the current timezone offset, regardless of DST
-        today (minus (t/today-at-midnight) time-zone-offset)
-        yesterday (plus today (days -1))]
+(defn- to-str
+  [ms old-fmt-fn yesterday-fmt-fn today-fmt-fn]
```

Comment:
this comment is a bit bizzarre:
```; this is wrong, it uses the current timezone offset,```
can you check who created this comment and have them create a bug on Github to address this if needs be.

Reply:
It's added in this commit https://github.com/status-im/status-mobile/blob/3c43a622e1b48733a60a7844afe5c8e8f858600d/src/status_im/utils/datetime.cljs#L49-L57
@cammellos is this comment still the case, should we create an issue tracking this?

Reply:
This one too. Should we fix this now? @cammellos https://github.com/status-im/status-mobile/blob/dd5c30b7a0ecbecf934917f7437eec9840b6c12c/src/status_im/chat/models/message_list.cljs#L185-L190

Reply:
> This one too. Should we fix this now? @cammellos https://github.com/status-im/status-mobile/blob/dd5c30b7a0ecbecf934917f7437eec9840b6c12c/src/status_im/chat/models/message_list.cljs#L185-L190

I'd say it's not worth it, maybe best to leave a comment, basically what it does, it takes the offset of the timezone as **today** to calculate a date **in the past**. 

In theory the date should be localized within the timezone offset that it was posted on, I don't think it's a big deal to be honest, not worth fixing, but maybe worth commenting in case someone notices offset timestamps/dates (it's a bit of an edge case).

Reply:
I've changed these two `TODO` to `NOTE(performance)` and `NOTE(edge-case)`
The performance one will be spotted easily if there's performance issue in the future.
I'll leave some keywords in this PR for the timezone one for future search.

invalid date time travel moved new timezone offsite message dst utc